server/http_server.py

```
@@ -0,0 +1,493 @@
+    return input_schema
+
+
+def get_version() -> str:
+    """Read version from VERSION.txt file."""
+    try:
+        # Get the project root directory (parent of server directory)
+        project_root = Path(__file__).parent.parent
+        version_file = project_root / "VERSION.txt"
+        
+        if version_file.exists():
+            with open(version_file, 'r') as f:
```

Ruff (0.12.2)

Unnecessary mode argument

Remove mode argument

(UP015)